crates/meroctl/src/cli/app.rs

```diff
@@ -51,6 +52,21 @@ impl Report for Application {
             println!("  {item:?}");
         }
     }
+
+    fn pretty_report(&self) {
+        let mut table = Table::new();
+        let _ = table.set_header(vec![Cell::new("Application Details").fg(Color::Blue)]);
+        let _ = table.add_row(vec![format!("ID: {}", self.id)]);
+        let _ = table.add_row(vec![format!("Size: {}", self.size)]);
+        let _ = table.add_row(vec![format!("Blob ID: {}", self.blob)]);
+        let _ = table.add_row(vec![format!("Source: {}", self.source)]);
+        let _ = table.add_row(vec!["Metadata:"]);
+
+        for item in &self.metadata {
+            let _ = table.add_row(vec![format!("  - {item:?}")]);
+        }
+        println!("{table}");
```

Comment:
Is there any value to printing a single column as a table? different from what we already had?

At least break up the "type" and "data" into adjacent columns instead of merging them in the same cell.